seacrowd/sea_datasets/uit-viic/uit-viic.py

```diff
@@ -0,0 +1,161 @@
+            }
+            """
+
+_DESCRIPTION = """
+            UIT-ViIC contains manually written captions for images from Microsoft COCO dataset relating to sports
+            played with ball. UIT-ViIC consists of 19,250 Vietnamese captions for 3,850 images. For each image,
+            UIT-ViIC provides five Vietnamese captions annotated by five annotators.
+            """
+
+_HOMEPAGE = "https://drive.google.com/file/d/1YexKrE6o0UiJhFWpE8M5LKoe6-k3AiM4"
+_PAPER_URL = "https://arxiv.org/abs/2002.00175"
+_LICENSE = Licenses.UNKNOWN
+_HF_URL = ""
+_SUPPORTED_TASKS = [Tasks.IMAGE_CAPTIONING]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "0.0.0"
+
+_UIT_VIIC_URL = "https://drive.google.com/uc?export=download&id=1YexKrE6o0UiJhFWpE8M5LKoe6-k3AiM4"
+_Split_Path = {
+    "train": "UIT-ViIC/uitviic_captions_train2017.json",
+    "validation": "UIT-ViIC/uitviic_captions_val2017.json",
+    "test": "UIT-ViIC/uitviic_captions_test2017.json",
+}
+
+
+class UITViICDataset(datasets.GeneratorBasedBuilder):
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(name="uit-viic_source", version=datasets.Version(_SOURCE_VERSION), description=_DESCRIPTION, subset_id="uit-viicd", schema="source"),
+        SEACrowdConfig(name="uit-viic_seacrowd_imtext", version=datasets.Version(_SOURCE_VERSION), description=_DESCRIPTION, subset_id="uit-viic", schema="seacrowd"),
+    ]
+
+    def _info(self):
+        if self.config.schema == "source":
+            return datasets.DatasetInfo(
+                # This is the description that will appear on the datasets page.
+                description=_DESCRIPTION,
+                # datasets.features.FeatureConnectors
+                # {"license": 2, "file_name": "000000535668.jpg",
+                #  "coco_url": "http://images.cocodataset.org/train2017/000000535668.jpg", "height": 426, "width": 640,
+                #  "date_captured": "2013-11-19 18:06:56",
+                #  "flickr_url": "http://farm4.staticflickr.com/3061/2442277392_49eb08cb2f_z.jpg", "id": 535668}
+                # {"image_id": 157656, "caption": "Người đàn ông đang đánh tennis ngoài sân.", "id": 4990}
+                features=datasets.Features(
+                    {
+                        "license": datasets.Value("int32"),
+                        "file_name": datasets.Value("string"),
+                        "coco_url": datasets.Value("string"),
+                        "flickr_url": datasets.Value("string"),
+                        "height": datasets.Value("int32"),
+                        "width": datasets.Value("int32"),
+                        "date_captured": datasets.Value("string"),
+                        "image_id": datasets.Value("int32"),
+                        "caption": datasets.Value("string"),
+                        "cap_id": datasets.Value("int32"),
+                    }
+                ),
+                supervised_keys=None,
+                homepage=_HOMEPAGE,
+                citation=_CITATION,
+            )
```

Comment:
Please refer to the template on DatasetInfo format. Same for the seacrowd_imtext schema

> datasets.DatasetInfo(
>             description=_DESCRIPTION,
>             features=features,
>             homepage=_HOMEPAGE,
>             license=_LICENSE,
>             citation=_CITATION,
>         )